### PR TITLE
release 0.0.1-rc.32-v4

### PR DESCRIPTION
**Pull request by ben-kaufman**

### CODE REVIEW

File: migrate-dao.js [hunk 1]
```diff
@@ -679,6 +680,26 @@ async function migrateDAO ({ arcVersion, web3, spinner, confirm, opts, migration
         await logTx(tx, `${standAlone.name} initialized.`)
       }
 
+      if (standAlone.runFunctions !== undefined) {
```
Comment (orenyodfat) [inline migrate-dao.js]: can you add that to readme?

Comment (ben-kaufman) [reply migrate-dao.js]: Ok

**Comment by orenyodfat** [comment]
can we add standalone contract to https://github.com/daostack/migration/blob/master/migration-params.json , so it will be used as a reference ? @ben-kaufman 

does the log output the standalone contracts name and address ?

**Comment by ben-kaufman** [comment]
Yes it logs it. Added to the readme.

**Pull request merged**